### PR TITLE
 Fix test suite bug 8234332 

### PR DESCRIPTION
**Pull request by yazeng**
Modify RSVD detection.

### CODE REVIEW

**Comment by yazeng** [comment]
@ProtoTest is added to the review. #Closed

File: TestSuites/FileServer/src/SMB2Model/TestSuite/ReplayCreateDurableHandleV2PersistentTestCase.cs [hunk 1]
```diff
@@ -19875,6 +19875,7 @@ public partial class ReplayCreateDurableHandleV2PersistentTestCase : PtfTestClas
         #endregion
         
         #region Test Starting in S1675
+        [Microsoft.VisualStudio.TestTools.UnitTesting.Ignore]
```
Comment (chenlu0616) [inline TestSuites/FileServer/src/SMB2Model/TestSuite/ReplayCreateDurableHandleV2PersistentTestCase.cs]: Why ignore these test cases?

File: TestSuites/FileServer/src/SMB2/TestSuite/DurableHandle/DurableHandle.cs [hunk 2]
```diff
@@ -185,19 +185,6 @@ public void DurableHandleV1_Reconnect_AfterServerDisconnect()
             #endregion
         }
 
-        //
-        // [TestMethod]
-        // Exclude this test case It fails on 2012R2-Domain-NonCluster and 2012R2-Workgroup env,
-        // but succeed in 2012R2-Domain-Cluster. Root case is not found. But this case only has impact to 2012R2 env
-        //
```
Comment (chenlu0616) [inline TestSuites/FileServer/src/SMB2/TestSuite/DurableHandle/DurableHandle.cs]: Root cause. Besides, what about the 2016 RTM behavior?

File: TestSuites/FileServer/src/SMB2Model/TestSuite/ReplayCreateDurableHandleV2PersistentTestCase.cs (hunk 1 above)
Comment (yazeng) [inline TestSuites/FileServer/src/SMB2Model/TestSuite/ReplayCreateDurableHandleV2PersistentTestCase.cs]: Those cases are not good for testing Replay. A tempary solution is to remove them from the test suite.

**Pull request merged**